package.json

```diff
@@ -25,7 +25,8 @@
     "build-storybook": "SKU_TELEMETRY=false sku build-storybook",
     "deploy": "node scripts/deploy.js",
     "post-commit-status": "node scripts/postCommitStatus.js",
-    "release": "pnpm run setup && pnpm build-codemod && changeset publish && pnpm build && pnpm deploy",
+    "release": "pnpm run publish-package && changeset publish && pnpm build && pnpm deploy",
+    "publish-package": "pnpm run setup && pnpm build-codemod",
```

Comment:
Naming might be a bit misleading here. Thoughts on renaming this script to prepare-publish or similar? Given its not actually doing the publishing itself.